.github/workflows/ci.yml

```diff
@@ -0,0 +1,95 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - 'master'
+
+
+jobs:
+
+  lint:
+    name: Linter
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
```

Comment:
Probably worth changing this to the latest release, so demos can use the most up-to-date features.

Reply:
I'd personally also remove the `3.6` from the `name` line, so there are less changes needed in future revisions.

Reply:
Sure 🚀, less change is always better.